File: Lancement d'un programme exec/exo4.py
import os
import sys

def main():
    while True:
        dest = input("Répertoire destination (ou 'quitter') : ")

        if dest == "quitter":
            break

        cmd = f"cp -r . {dest}"
        os.system(cmd)

if __name__ == "__main__":
    main()

File: Lancement d'un programme exec/exo4.c
#include <stdio.h>
#include <stdlib.h>
#include <string.h>

int main() 
{
    char dest[256], cmd[512];

    while (1) 
    {
        printf("Répertoire destination (ou 'quitter') : ");
        scanf("%s", dest);

        if (strcmp(dest, "quitter") == 0) 
        {
            break;
        }

        
        sprintf(cmd, "cp -r . %s", dest);
        system(cmd);
    }

    return 0;
}